### PR TITLE
Bgs gradients

### PR DESCRIPTION
**Pull request by MahmoudBakr23**
Adding and styling the following:
- header
- navbar
- header-text
- main
- footer

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+nav {
+  position: relative;
+  top: 15%;
+  left: 17%;
+  background: #333 linear-gradient(180deg, #999, #333);
+  border: 0.1px solid #333;
+  border-radius: 5px;
+  width: 65%;
+  height: 7%;
+}
```
Comment (HADDADSOHAIB) [inline main.css]: The requirements in this project is to use flex, grid or float, I believe you can achieve a good result with this element in this example, please change the solution from using position: relative to flex, grid or float.
- [ ] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) and  [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

Comment (HADDADSOHAIB) [reply main.css]: If you need position: relative to make position: absolute work in a child element, you can keep it, but the positioning of the element should be with flex, grid or float.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  padding: 5px 5px;
+}
+
+.header-text {
+  position: absolute;
+  top: 30%;
+  left: 20%;
+  text-align: center;
+  width: 60%;
+}
```
Comment (HADDADSOHAIB) [inline main.css]: The same here for positioning using flex, grid or float.

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @MahmoudBakr23  and @OlawaleJoseph    :handshake: 

### Good job so far :clap:  your project looks like the original :medal_sports: 

### there are some changes to get the project approved :point_down:
- Please fix the size of the search input:
Screenshot from 2020-06-10 12-18-48
- The nav elements should be a link:
- [ ] The first eight elements should be links (to other sections or `#`), and the search bar should be an `<input>` tag ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))
- All the images ( 4 images ) should be also a link (check the original):
- [ ] It contains four elements which are links (to other articles or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))
- The background image should have background-size set to cover so it will take the full width in all sizes:
- [ ] The CSS background should `cover` the whole page irrespective of the screen size. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/background_size_requirement.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

### Once you are done with the requirements ask for another code review.

**Comment by MahmoudBakr23** [comment]
The required changes are done, Thank you!
For the re-sizing of the search bar, please see the screenshot attached to the README.md file.

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @MahmoudBakr23  and @OlawaleJoseph    :handshake: 

### Good job :clap:  you implemented almost every thing :medal_sports: however there is still some issues:
- I am not sure if you saw my comments:
Screenshot from 2020-06-10 13-55-42
Screenshot from 2020-06-10 13-55-52
This is a mandatory requirement, please work on it.
### Your are doing great work keep it up :+1:

File: main.css [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+  height: 85vh;
+  border: 1px solid #333;
+  background-image: url("./images/apple_background.jpg");
+  background-size: cover;
+  background-position: 50% 100%;
+}
+
+nav {
+  position: flex;
+  margin: 7% auto;
```
Comment (HADDADSOHAIB) [inline main.css:18]: `Position: flex;`
- [optional]flex is not a valid value for position, it doesn't get rendered in the browser:
Screenshot from 2020-06-10 14-34-08
what doing the positioning in your css here is the margin. so you can go on and delete all position: flex they don't serve any value.
- You are using display: flex in other places so you meet the requirements.

**Review by HADDADSOHAIB — APPROVED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Project Approved :+1: 

### Hello @MahmoudBakr23  and @OlawaleJoseph    :handshake: 

### Check my comments.
### You can merge this branch into master.

**Pull request merged**